submissions/description/sensitive_data_exposure/sensitive_token_in_url/template.md

```diff
@@ -0,0 +1,20 @@
+# Sensitive data disclosure: Sensitive Token in URL
+
+## Overview of the Vulnerability
+
+Sensitive data can be exposed when it is not behind an authorization barrier. When this information is exposed it can place the application at further risk of compromise. The application discloses a sensitive token in the URL which, if captured by an attacker, can be used to escalate privileges or authorize API calls within the application.
```

Comment:
Technical wording change will help explain the impact here

> Sensitive data can be exposed when it is not properly transported or secured during transmission, When this information is exposed it can place the application at further risk of compromise. The application discloses a sensitive token in the URL, which, if captured by an attacker, can be used to gain access to the users account through this token, breaching the Confidentiality and Integrity of that account.